### PR TITLE
add declarations for unidecode

### PR DESCRIPTION
**Pull request by vfernandestoptal**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/unidecode/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+// Definitions by: Vitor Fernandes <https://github.com/vfernandestoptal>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ Note that ES6 modules cannot directly export callable functions.
+ *~ This file should be imported using the CommonJS-style:
+ *~   import x = require('someLibrary');
+ *~
+ *~ Refer to the documentation to understand common
+ *~ workarounds for this limitation of ES6 modules.
+ */
```
Comment (plantain-00) [inline types/unidecode/index.d.ts]: The comments can be removed

File: types/unidecode/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Type definitions for unidecode 0.1
+// Project: https://github.com/FGRibreau/node-unidecode
+//          http://blog.fgribreau.com/2012/05/unidecode-for-javascript-nodejs.html
```
Comment (plantain-00) [inline types/unidecode/index.d.ts]: This link seems irrelevant, it should be the project url, rather than a blog url

**Review by plantain-00 — CHANGES_REQUESTED** [review]

**Comment by vfernandestoptal** [comment]
requested changes done

**Review by plantain-00 — APPROVED** [review]
